src/elements/common/sub-header/ViewModeChangeButton.scss

```diff
@@ -1,4 +1,4 @@
-.bdl-ViewModeChangeButton {
+.btn.bdl-ViewModeChangeButton {
```

Comment:
same approach used for `AddButton` and `SortButton`: https://github.com/box/box-ui-elements/blob/master/src/elements/common/sub-header/AddButton.scss#L1